### ISSUE TITLE
C Modify Friend/Helper List

### ISSUE THREAD

**Issue by Spazgazum** [drewhoener/JWSuggestions#18]
Hi.

I had this idea the other day and thought others might enjoy it too.
I suggest that we add an option for a list(s) that we could add to the cmodify command for stores or people that play with multiple friends where we could make one list (or a few if possible) where we can save specific names to for quick adding many to locked items.

Ex. I like to play with four other friends.
/cmodify editlist friends add Sarabi_Mami
and so on for the others.
Then next time I make a chest i just put /cmodify Friends and everyone would be added.

Benefits:

Promotes community and friendship. : ) 

Would help shops with multiple helpers because owners could add helpers to boxes much quicker and easier considering shops have to share access with with a plethora chests.

Thanks for your consideration. -Spaz

**Comment by SonOfDeath** [comment]
That's a good idea :)

For now (in case you didn't know this) you can type in the command once /cmodify (name)

Then if you want to do it again go like you would insert a message and hit the up arrow the command will re appear again for you to use.

**Comment by SarabiMami** [comment]
Lol We do know know to slash up, but thanks for bringing it up for those who don't know. 
We have four friends we have to add to everything, this gets to be a very long task with chests, slash up helps a bit, but it would be so nice to have a list

**Comment by Spazgazum** [comment]
Yeah. Thanks. I know (and use) the / up buttons all the time, but it's good for other people to read so they can know about it.

I really just mean for adding multiple different names. Me and my friends are big on mining and redstone contraptions, so there's chests and droppers, hoppers, dispensers, etc. Instead of walking down a line of hoppers slash+up with one persons nname, then the same for 3+ other people, we could use this to add them all at once. 

Maybe I'm just lazy lol. Nah really my main concern is sometimes we miss one person or accidentally add someone twice instead of another person and not realize it until they can't access it when needed.

**Comment by drewhoener** [comment]
Actually I've been considering a friend system for a while now, currently in dev for the new version. I like the idea of adding all at once. My original intent was that you would be able top Set separate flags for friends than for others

**Comment by Spazgazum** [comment]
Awesome.

**Comment by Duranduran05orDuran** [comment]
+1 support

**Comment by beardobob** [comment]
I have probably 100+ chests with hoppers, it would be nice to have an add person to all chests for a store that me and son are making instead of going to each chest hopper and door,   maybe /cmodify playername all to unlock all chests etc
